pkg/test/framework/components/echo/match/matchers.go

```diff
@@ -52,116 +50,118 @@ func Or(ms ...Matcher) Matcher {
 }
 
 // Not negates the given matcher. Example:
-//     Not(IsNaked())
+//     Not(Naked())
 func Not(m Matcher) Matcher {
 	return func(i echo.Instance) bool {
 		return !m(i)
 	}
 }
 
-// ServicePrefix matches instances whose service name starts with the given prefix.
-func ServicePrefix(prefix string) Matcher {
+// SameService matches instances with the given namespace and service name.
+func SameService(n model.NamespacedName) Matcher {
```

Comment:
nit: SameService was a confusing name to me. Maybe rename Service to ServiceName and name this one Service?